WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -159,7 +159,33 @@ public async Task<Wallet> StartWalletAsync(Wallet wallet)
 				await wallet.StartAsync(cancel).ConfigureAwait(false);
 				Logger.LogInfo($"Wallet '{wallet.WalletName}' started.");
 				cancel.ThrowIfCancellationRequested();
-
+				
+				// Continue wallet synchronization in the background for all keys skipped by TurboSync.
+				_ = Task.Run(
```

Comment:
You can move this whole task into the `wallet.StartAsync` and fire the task there.